telnyx/api_resources/sim_card.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, division, print_function
+
+from telnyx.api_resources.abstract import (
+    ListableAPIResource,
+    UpdateableAPIResource,
+    nested_resource_class_methods,
+)
+
+
+@nested_resource_class_methods(
+    "activate", path="actions/activate", operations=["create"]
+)
+@nested_resource_class_methods(
+    "deactivate", path="actions/deactivate", operations=["create"]
+)
+@nested_resource_class_methods(
+    "register", path="/v2/actions/register/sim_cards", operations=["create"]
```

Comment:
This is not a nested resource method, which means it breaks the use of `nested_resource_class_methods`. However, I thought this would be a much nicer solution than having a separate 'GlobalActiones' type class to store this action instead.

Reply:
hmmm.. Just wondering why this URI was created in this way at first. It sounds like a nested resource.
Anyways, I agree this solution is better than have a new GlobalActiones

Reply:
@luizanao this URI was used as it's an action on the collection as opposed to an action on a specific instance (`./sim_cards/:id`). This is noted in our API design guidelines under Requests > Actions